src/background.js

```diff
@@ -747,16 +748,25 @@ function startBackgroundListeners() {
  * lets get this party started
  */
 console.log('Loading badgers into the pen.');
+var badger;
+window.legacyStorage = null; 
+var legacyKeys = ["cookieDb","domainExceptionSites","userRed","userYellow","userGreen","disabledSites","disabledSitesPrivate","domainExceptions","domainExceptionsPrivate","seenComic","changedCookies","preloads","originFrequency","originFrequencyPrivate","blockedOrigins","policyHashes","policyWhitelist","nextBlockedSitesCheck"];
+chrome.storage.local.get(legacyKeys, function(legacy){
```

Comment:
Do we need to do this out here, instead of inside `src/migrations.js`? Cooper I remember you mentioned something wasn't working when you tried to retrieve legacy storage keys inside the migrations file but why?